xapian-applications/omega/omega.cc

```diff
@@ -360,6 +365,10 @@ try {
     for (auto i = begin; i != end; ++i) {
 	const string & v = i->second;
 	if (!v.empty()) {
+	    if (i->first[CONST_STRLEN("START.")] < '0' ||
+		i->first[CONST_STRLEN("START.")] > '9') {
```

Comment:
There's a `C_isdigit()` function (in `stringutils.h`) which is essentially the same as the standard `isdigit()` except that it's hard-wired to the C locale so will only ever report true for ASCII '0' to '9' - using that would result in clearer code with less repetition:

```
    if (!C_isdigit(i->first[CONST_STRLEN("START.")])) {
```

Reply:
thanks i will fix that.